docs/topic-guides/hp-tuning-det/hp-single.txt

```diff
@@ -3,8 +3,8 @@
 Hyperparameter Search: Single
 =============================
 
-The ``single`` search method does not technically "search": it trains a
-single hyperparameter configuration for ``max_steps`` steps, and then
+The ``single`` search method does a very trivial "search": it trains a
```

Comment:
the least blocking of comments: Does this make us sound bad? Maybe "does a simple 'search'"?

Reply:
That's the mathematician in me, I guess. I think "very minimal" would have the right effect.